api/ceramic.yaml

```diff
@@ -101,41 +101,6 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
-  /events/{event_id}:
```

Comment:
why removing this endpoint?  It's used in some of our test currently so removing it will break those tests